sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/PubsubLiteReadSchemaTransformProvider.java

```diff
@@ -118,12 +166,14 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
                                         .build())
                                 .build()))
                     .apply(
-                        MapElements.into(TypeDescriptors.rows())
-                            .via(
-                                seqMess ->
-                                    valueMapper.apply(
-                                        seqMess.getMessage().getData().toByteArray())))
-                    .setRowSchema(beamSchema));
+                        ParDo.of(new ErrorFn("Read-Error-Counter", valueMapper))
```

Comment:
Can you change the metric name to use format "IO name"-"read/write"-error-counter. Something like PubSubLite-read-error-counter Keeping the same format makes it a bit easier on me later to capture them in our backend.

Reply:
Done